rebasehelper/helpers/copr_helper.py

```diff
@@ -54,18 +54,27 @@ def get_client(cls):
             return client
 
     @classmethod
-    def create_project(cls, client, project, chroot, description, instructions, permanent=False, hide=True):
+    def create_project(cls, client, project, chroots, description, instructions, permanent=False, hide=True):
         try:
             client.project_proxy.add(ownername=client.config.get('username'),
                                      projectname=project,
-                                     chroots=[chroot],
+                                     chroots=chroots,
                                      delete_after_days=None if permanent else cls.DELETE_PROJECT_AFTER,
                                      unlisted_on_hp=hide,
                                      description=description,
                                      instructions=instructions)
-        except CoprRequestException:
-            # reuse existing project
-            pass
+        except CoprRequestException as e:
+            error = e.result.error
+            try:
+                [[error]] = error.values()
+            except AttributeError:
+                pass
+
+            if error.startswith('You already have project named'):
+                # reuse existing project
+                pass
+            else:
+                raise RebaseHelperError('copr: ' + error)
```

Comment:
Ok, but this is not very user friendly. Why not raise something like [this](https://github.com/rebase-helper/rebase-helper/blob/ec13174dda28800825354110b88fe45f78556b35/rebasehelper/specfile.py#L296-L297)?